# Lista de Estrutura Sequencial

#### 1. Faça um Programa que mostre a mensagem (print) "Alo mundo" na tela.

In [ ]:
print('Olá mundo')

#### 2. Faça um Programa que peça um número (input) e então mostre a mensagem: "O número informado foi [número]."

In [3]:
n=int(input('Digite um número '))
print(f'O numeor digitado foi {n} ')

Digite um número 98


NameError: name 'pritn' is not defined

#### 3. Faça um Programa que peça dois números e imprima a soma.

In [1]:
n=int(input('Digite um número '))
n2=int(input('Digite outro numero '))

print(f'A soma dos numeros é {n+n2}')

Digite um número 8
Digite outro numero 3
A sima dos numeros é 11


#### 4. Faça um Programa que peça as 4 notas bimestrais de um aluno e mostre a média de todas as notas.

In [ ]:
n=float(input('Digite um número '))
n2=float(input('Digite outro numero '))
n3=float(input('Digite um número '))
n4=float(input('Digite outro numero '))
print(f' A media do aluno foi {(n+n2+n3+n4)/4}')

#### 5. Faça um Programa que converta metros para centímetros. Você pode pedir o comprimento em metros para o usuário (input).

In [7]:
med=float(input('Digite uma meidida em metros '))
converter=med*100
print(f'{med} metrso convertendo para cm fica {converter} cm ')

Digite uma meidida em metros 2
2.0 metrso convertendo para cm fica 200.0


#### 6. Faça um Programa que calcule a área de uma sala de um apartamento. Para isso, o seu programa precisa pedir a largura da sala, o comprimento da sala e imprimir a área em m² da sala.

In [8]:
larg=50
comp=45
area=larg*comp
print(f'A area da sala é de {area} m²')

A area da sala é de 2250 m²


#### 7. Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês. Calcule e mostre o total do seu salário no referido mês.

#### 8. Vamos criar um conversor de temperatura. Faça um Programa que peça a temperatura em graus Fahrenheit, transforme e mostre a temperatura em graus Celsius.
$C = \frac{5}{9}(F-32)$

#### 9. Faça um Programa que peça a temperatura em graus Celsius, transforme e mostre em graus Fahrenheit.
$F = \frac{9}{5}C + 32$

#### 10. Tendo como dados de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, usando a seguinte fórmula:
$P = 72,7h - 58$

Lembrando que "algoritmo" nada mais é do que um programa, como todos os outros que você vem fazendo

#### 11. Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:
##### a. Para homens: $P = 72,7h - 58$
##### b. Para mulheres: $P = 62,1h - 44,7$

#### 12. Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês.

#####  Calcule o salário bruto (horas * salario por hora)

##### Calcule o desconto do IR (11% do salário bruto)

##### Calcule o desconto do INSS (8% do salário bruto)

##### Calcule o desconto do sindicato (5% do salário bruto)

##### Calcule o salário líquido (salário bruto - descontos)

#### 13. Faça um programa para uma loja de tintas. O programa deverá pedir o tamanho em metros quadrados da área a ser pintada. Considere que a cobertura da tinta é de 1 litro para cada 3 metros quadrados e que a tinta é vendida em latas de 18 litros, que custam R\$ 80,00. Informe ao usuário a quantidades de latas de tinta a serem compradas e o preço total. (para simplificação nesse momento, não se preocupe em arredondar a quantidade de latas a serem compradas - vamos trabalhar isso em breve)

#### 14. Faça um programa que peça o tamanho de um arquivo para download (em MB) e a velocidade de um link de Internet (em Mbps), calcule e informe o tempo aproximado de download do arquivo usando este link (em minutos).

Detalhe: MB significa megabyte, Mb (com b minúsculo) significa megabit. Um megabit é 1/8 de um megabyte. 